src/modules/getBullsAndCows.js

```diff
@@ -13,7 +13,22 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const userInputArr = userInput.toString().split('');
+  const numberToGuessArr = numberToGuess.toString().split('');
+  const result = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  userInputArr.forEach((el, i) => {
+    if (el === numberToGuessArr[i]) {
+      result.bulls++;
+    } else if (numberToGuessArr.includes(el)) {
+      result.cows++;
+    }
+  });
```

Comment:
Consider using a different strategy to improve the performance of this code. For instance, you could create a map (an object in JavaScript) to store the count of each digit in `numberToGuessArr`. Then, for each digit in `userInputArr`, you can check if it's in the map and update the counts of bulls and cows accordingly. This will reduce the time complexity to linear.